crates/findex/src/cloud.rs

```diff
@@ -465,11 +465,17 @@ impl FindexCallbacks<FindexCloudError, UID_LENGTH> for FindexCloud {
         Ok(true)
     }
 
+    async fn fetch_all_entry_table_uids(
+        &self,
+    ) -> Result<HashSet<Uid<UID_LENGTH>>, FindexCloudError> {
+        todo!("fetch all entry table uids not implemented in WASM")
```

Comment:
Because we didn't implement the compact in Findex Cloud. Right now the compact operation is under a massive rewrite so I think we'll pause the implementation to wait for the crypto team to decide the correct implementation.

Reply:
I think this function is used for compacting only